pkg/internal/diskutil.go

```diff
@@ -0,0 +1,338 @@
+package internal
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"golang.org/x/sys/unix"
+)
+
+const (
+	// StateSuspended is a possible value of BlockDevice.State
+	StateSuspended = "suspended"
+	// DiskByIDDir is the path for symlinks to the device by id.
+	DiskByIDDir = "/dev/disk/by-id/"
+)
+
+var (
+	// DefaultLockTimeout to use for WaitForPVCreationLock
+	DefaultLockTimeout = time.Second * 30
+	// DefaultLockInterval to use for
+	DefaultLockInterval = time.Second
```

Comment:
is anything using these values?

Reply:
removed, commited fix, will be in next push